src/benchmarks/micro/corefx/System.Xml.Linq/Perf.XElement.cs

```diff
@@ -0,0 +1,56 @@
+﻿using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+
+namespace System.Xml.Linq
+{
+    [BenchmarkCategory(Categories.CoreFX)]
+    public class Perf_XElement
+    {
+        private XElement _element;
+
+        [Benchmark]
+        public XElement CreateElement() => new XElement("Root", "text node");
+
+        [Benchmark(OperationsPerInvoke = 8)]
+        public XElement CreateWithElements()
+        {
+            XElement doc = new XElement("Root",
+                new XElement("elem1", "text node two e1; text node three"),
+                new XElement("elem1", "text node two e1; text node three"),
```

Comment:
> "text node two e1; text node three"

This sounds like it makes several nodes but it simply creates `<elem1>text node two e1; text node three</elem1>`

A string like "some xml element content" would be clearer.